_includes/2020/ImageRandomizer.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Keyman\Site\com\keyman;
+
+class ImageRandomizer {
+    
+    static function randomizer($folder) { 
+        if(!empty($_SERVER['DOCUMENT_ROOT']) && !empty($folder)) {
+          $imgDir = $_SERVER['DOCUMENT_ROOT'] . '/cdn/dev/' . $folder;
+          $img = glob($imgDir . '*.{png,jpg,jpeg,gif}', GLOB_BRACE);
+          if(!empty($img)) {
+            shuffle($img);
+            return str_replace($_SERVER['DOCUMENT_ROOT'], '',  $img[0]);
+          }
+        }
+        return "Not found";
+      }
+}
+
+?>
```

Comment:
This along with the png's will be removed once keymanapp/shared-sites#21 is approved and merged.

It will affect the use / require_once paths below too.

Reply:
It's now merged and available in shared-sites v0.9

If you change this line to v0.9
https://github.com/keymanapp/keyman.com/blob/c47b0ed397fc766a3ab1272ae2601984bb7f6a12/build.sh#L5

and run `./build.sh configure`, it should pull in the image randomizer.

Reply:
Yes, just the version number

Reply:
`./build.sh configure` means you can also remove the ImageRandomizer.php file and all the .png files on this PR.